RELEASING.md

```diff
@@ -191,23 +193,32 @@ at the same time that subcomponent release branches are cut, often well before t
 1. Create a new branch off of the latest master.
 
    ```
-   git checkout -b release-candidate-vX.Y
-   ```
-
-1. In [netlify.toml](netlify.toml), set the `CANDIDATE_RELEASE` environment variable:
-
-   ```toml
-   [build.environment]
-     CANDIDATE_RELEASE = "vX.Y"
+   git checkout -b release-vX.Y
    ```
-
+   
+1. Enable branch deployments for this branch on netlify by adding branch name to `Deploy contexts` in site settings.
+   
 1. Commit your changes. For example:
 
    ```
    git commit -m "build vX.Y candidate"
    ```
+   
+1. Open a pull request to the upstream release-vX.Y branch. Get it reviewed and merged. 
+
```

Comment:
There are no changes anymore since we no longer update netlify.toml above, so there's nothing to commit. Can this be removed?

Reply:
sadly..we need a commit here to trigger the build (since the candidate branch is just added to branch deploys). I'm planning to ask the user to change the `RELEASE_VERSION` in the `netlify.toml` instead